src/components/home/hero_section.js

```diff
@@ -21,7 +26,13 @@ const renderParallaxImage = ({ baseDelay, image, parallaxAmount }) => (
               <Img
                 fadeIn={false}
                 onLoad={onLoad}
-                fluid={image}
+                fluid={[
+                  mobileImage,
```

Comment:
Doesn't the mobile image need a limiting media rule like for team member images?